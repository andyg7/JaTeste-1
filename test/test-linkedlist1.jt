#include_jtlib <int_list.jt>

func int main()
{

	struct int_list *my_list;
	my_list = initialize();
	insert(my_list,9);
	insert(my_list,5);
	insert(my_list,8);
	insert(my_list,10);
	insert(my_list,40);
	insert(my_list,11);
	insert(my_list,0);
	insert(my_list,9);
	insert(my_list,478);
	print_list(my_list);

	return 0;
}
