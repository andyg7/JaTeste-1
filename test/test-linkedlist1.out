9
5
8
10
40
11
0
9
478
